<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-C57COV6" failures="1" tests="2" name="Prueba10" time="437.231" errors="0" timestamp="20 jul. 2021 20:31:43 GMT">
  <testcase classname="com.timp.test.MDR.ParametrosContabilizacao.MapeamentoContabil.MapeamentoContabilExcluirEmMassa" name="criar" time="345.34">
    <failure type="java.lang.AssertionError" message="O registro não foi criado com sucesso expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: O registro não foi criado com sucesso expected [true] but found [false]
at com.timp.test.MDR.ParametrosContabilizacao.MapeamentoContabil.MapeamentoContabilExcluirEmMassa.criar(MapeamentoContabilExcluirEmMassa.java:42)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- criar -->
  <testcase classname="com.timp.test.MDR.ParametrosContabilizacao.MapeamentoContabil.MapeamentoContabilExcluirEmMassa" name="Excluir" time="0.0">
    <skipped/>
  </testcase> <!-- Excluir -->
</testsuite> <!-- Prueba10 -->
